bool detected
uint16 position_x
uint16 position_y
